editor/plugins/node_3d_editor_plugin.cpp

```diff
@@ -9275,7 +9274,7 @@ Node3DEditor::Node3DEditor() {
 	sun_button = memnew(Button);
 	sun_button->set_tooltip_text(TTRC("Toggle preview sunlight.\nIf a DirectionalLight3D node is added to the scene, preview sunlight is disabled."));
 	sun_button->set_toggle_mode(true);
-	sun_button->set_accessibility_name(TTRC("Preview Sunlight"));
+	sun_button->set_accessibility_name(TTRC("Toggle preview sunlight."));
```

Comment:
For these kinds of accessibility descriptions where the difference is basically a new line, I wonder if you could have them borrow from the same source. For example (pseudocode):
```c++
String first_line = TTRC("Toggle preview sunlight.")
sun_button->set_accessibility_name(first_line)
sun_button->set_tooltip_text(first_line + "\n" + TTRC("If a DirectionalLight3D node is added to the scene, preview sunlight is disabled."))
```

It may be confusing for translations, though